RDF.js

```diff
@@ -1044,7 +1054,7 @@ function importItem(newItem, node) {
 	// description/attachment note
 	if(newItem.itemType == "attachment") {
 		newItem.note = getFirstResults(node, [n.dc+"description", n.dc1_0+"description", n.dcterms+"description"], true);
-	} else if (!newItem.abstractNote) {
+	} else if (!newItem.abstractNote && !isZoteroRDF) {
 		newItem.abstractNote = getFirstResults(node, [n.dc+"description", n.dcterms+"description"], true);
 	}
```

Comment:
Can we move the lines from above here, such that all cases for `dc:description` are handled in one place? Something like (not tested):
```js
 	// dc:description
 	if (newItem.itemType == "attachment") {
 		newItem.note = getFirstResults(node, [n.dc+"description", n.dc1_0+"description", n.dcterms+"description"], true);
 	} else if (isZoteroRDF) {
		// for ZoteroRDF dc:description holds the data from extra
		newItem.extra = newItem.extra ? newItem.extra + getFirstResults(node, [n.dc+"description"], true) : getFirstResults(node, [n.dc+"description"], true);
	} else if (!newItem.abstractNote) {
 		newItem.abstractNote = getFirstResults(node, [n.dc+"description", n.dcterms+"description"], true);
 	}
```
Maybe even test it directly here without an additional variable.